modin/backends/base/query_compiler.py

```diff
@@ -2548,8 +2548,8 @@ def view(self, index=None, columns=None):
         BaseQueryCompiler
             New masked QueryCompiler.
         """
-        index = [] if index is None else index
-        columns = [] if columns is None else columns
+        index = slice(None) if index is None else index
+        columns = slice(None) if columns is None else columns
```

Comment:
bug in the `BaseQueryCompiler`. We have to grab the whole axis if the indexer is None, previously it was taking no elements instead.